### PR TITLE
Move to Quarkus 3.0.0.Alpha3

### PR DESCRIPTION
**Pull request by rsvoboda**
Move to Quarkus 3.0.0.Alpha3

- switch to jakarta packages
- drop of Quarkiverse extensions with really old relocations
- remove commented out modules
- PR GH Action - make sure to also build required modules 
- skip native build and run as keycloak-authz-client is not native ready

Remove commented out modules ... no plans to rework them as the development happens in https://github.com/quarkus-qe/quarkus-test-suite

### CODE REVIEW

File: 005-quarkus-and-maven-profiles/pom.xml [hunk 1]
```diff
@@ -19,7 +19,7 @@
         <dependency>
             <groupId>io.quarkus</groupId>
             <artifactId>quarkus-spring-di</artifactId>
-            <version>1.3.2.Final-redhat-00001</version>
+            <version>3.0.0.Alpha3</version>
```
Comment (michalvavrik) [inline 005-quarkus-and-maven-profiles/pom.xml:22]: module compiles without specifing `3.0.0.Alpha3`, is there advantage in specifing it?

Comment (rsvoboda) [reply 005-quarkus-and-maven-profiles/pom.xml:22]: There is no 3.based redhat version yet. old versions are pulling in javax stuff
This change is just to make it run ...

Comment (michalvavrik) [reply 005-quarkus-and-maven-profiles/pom.xml:22]: tbh I don't comprehend. I thought we have beefy scenarios to run against 999-SNAPSHOT and when no version is specified, then `quarkus-spring-di:999-SNAPSHOT` is used and tests passes. please give me a hint that will help me to get it, thanks

Comment (rsvoboda) [reply 005-quarkus-and-maven-profiles/pom.xml:22]: redhat version is used to ensure RH maven repo got activated ...

There was issue in old Quarkus that this didn't work ... Quarkus 1.3 I guess

Comment (rsvoboda) [reply 005-quarkus-and-maven-profiles/pom.xml:22]: Maybe I should add a TODO note to move to -redhat when there is 3.x version released

Comment (michalvavrik) [reply 005-quarkus-and-maven-profiles/pom.xml:22]: ok

Comment (rsvoboda) [reply 005-quarkus-and-maven-profiles/pom.xml:23]: Added

**Review by michalvavrik — APPROVED** [review]
lgtm

**Comment by rsvoboda** [comment]
CI is green, PR approved, merging

**Pull request merged**